src/lib/io/ParticleIO.cpp

```diff
@@ -54,6 +54,8 @@ readers()
     static bool initialized=false;
     if(!initialized){
 	initializationMutex.lock();
```

Comment:
if we make these plugins opt-in then we'll probably need `#defines` that match them so that we can conditionally compile these lines below.